PKGBUILD

```diff
@@ -25,6 +25,10 @@ _dir="mavros-${pkgver}/libmavconn"
 source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/mavlink/mavros/archive/${pkgver}.tar.gz")
 sha256sums=('c7cd33fe3582c427744d251383b0befb0766ab29cd6191a3f29f6a439aa26813')
 
+prepare() {
+  sed -i '11s/Geographic)/GeographicLib)/' ${_dir}/cmake/Modules/FindGeographicLib.cmake
```

Comment:
The branch to make a PR against is `master`. ROS1 releases/tags follow the master branch for that project.